egs/wsj/s5/utils/lang/bpe/add_final_optional_silence.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+. ./path.sh
+
+final_sil_prob=0.5
+
+echo "$0 $@"  # Print the command line for logging
+
+. utils/parse_options.sh
+
+if [ $# -ne 1 ]; then
+  echo "Usage: $0  <lang>"
+  echo " Add final optional silence to lexicon FSTs (L.fst and L_disambig.fst) in  "
+  echo " lang/ directory <lang>."
```

Comment:
Please explain why this might bre useful, e.g.:
```This can be useful in systems with byte-pair encoded (BPE) lexicons, in which
 the word-initial silence is part of the lexicon, so we turn off the standard optional
 silence in the lexicon```